generator.yaml

```diff
@@ -17,6 +17,40 @@ operations:
     operation_type: Delete
     resource_name: HyperParameterTuningJob
 resources:
+  Pipeline:
+    fields:
+      PipelineStatus:
+        is_read_only: true
+        print:
+          name: STATUS
+        from:
+          operation: DescribePipeline
+          path: PipelineStatus
+      CreationTime:
+        is_read_only: true
+        from:
+          operation: DescribePipeline
+          path: CreationTime
+      LastModifiedTime:
+        is_read_only: true
+        from:
+          operation: DescribePipeline
+          path: LastModifiedTime
+      LastRunTime:
```

Comment:
if we have this, we need this resource to be requeueing continuously. lets skip for now or add reuqueueonsuccess?

Reply:
will skip for now